cohttp-lwt-unix/src/net.mli

```diff
@@ -33,6 +33,19 @@ val init : ?ctx:Conduit_lwt_unix.ctx -> ?resolver:Resolver_lwt.t -> unit -> ctx
     {!Resolver_lwt.t} with a different name resolution strategy (for instance to
     override a hostname to point it to a Unix domain socket). *)
 
+type endp = Conduit.endp
```

Comment:
Why are re-exporting this alias?

Reply:
because it is added in `Cohttp_lwt.S.Net` as abstract type that needs to be satisfied. I changed cohttp-unix/src/net.mli to `include Cohttp_lwt.S.Net with type endp = Conduit.endp` to deduplicate code and documentation.

The `endp` type is needed in `Connection_cache`.